Tools/jit/_targets.py

```diff
@@ -226,6 +235,13 @@ def build(
             request = "Please report any issues you encounter.".center(len(warning))
             outline = "=" * len(warning)
             print("\n".join(["", outline, warning, request, outline, ""]))
+
+        if self.llvm_version != _llvm._LLVM_VERSION:
+            warning = f"LLVM version {self.llvm_version} is not officially supported!"
+            request = "Use custom LLVM version at your own rist.".center(len(warning))
+            outline = "=" * len(warning)
+            print("\n".join(["", outline, warning, request, outline, ""]))
+
```

Comment:
Instead of making this it's own banner, can we just add the check and the warning into the existing banner? I'm thinking that we can replace

```python
request = "Please report any issues you encounter.".center(len(warning))
```

when we see the user is overriding the LLVM version. We can just print something like:

```
Warning! Building with an LLVM version other than {_llvm._LLVM_VERSION} is not supported.
```